app/script/view_model/content/PreferencesOptionsViewModel.js

```diff
@@ -72,7 +72,9 @@ z.viewModel.content.PreferencesOptionsViewModel = class PreferencesOptionsViewMo
 
   saveCallLogs() {
     const messageLog = this.callingRepository.messageLog;
-    if (messageLog.length) {
+    // Very short logs will not contain useful information
+    const logContainsCall = messageLog.length > 10;
+    if (logContainsCall) {
```

Comment:
maybe including count in the variable name.. what is this length actually represents?

Reply:
Adjusted it and got rid of the magic number.